contracts/PreseasonTwo/BuilderNFTPreSeason02Implementation.sol

```diff
@@ -396,6 +396,29 @@ contract BuilderNFTPreSeason02Implementation is
     function minter() external view returns (address) {
         return MemoryUtils._getAddress(MemoryUtils.MINTER_SLOT);
     }
+    function mintTo(
+        address account,
+        uint256 tokenId,
+        uint256 amount
+    ) external onlyAdminOrMinter {
+        _validateMint(account, tokenId);
+        _mintTo(account, tokenId, amount);
+    }
+
+    function _mintTo(
+        address account,
+        uint256 tokenId,
+        uint256 amount
+    ) internal {
+        // Mint tokens
+        BuilderNFTPreSeasonStorage.increaseBalance(account, tokenId, amount);
+
+        // Update total supply
+        BuilderNFTPreSeasonStorage.increaseTotalSupply(tokenId, amount);
```

Comment:
BuilderNFTPreSeasonStorage already handles total supply increase / decrease so that we don't forget to add it in when increasing balances